app/actions/handlers.py

```diff
@@ -0,0 +1,261 @@
+                on=httpx.HTTPError,
+                attempts=3,
+                wait_initial=datetime.timedelta(seconds=10),
+                wait_max=datetime.timedelta(seconds=10),
+        ):
+            with attempt:
+                aoi_data = await client.get_aoi_data(
+                    integration=integration,
+                    config=action_config
+                )
+
+                # Some AOIs do not have an associated Geostore so we short-circuit here a report in the logs.
+                if not aoi_data.attributes.geostore:
+                    msg = f"No Geostore associated with AOI {aoi_data.id}."
+                    logger.error(
+                        msg,
+                        extra={
+                            "needs_attention": True,
+                            "integration_id": str(integration.id),
+                            "aoi_id": aoi_data.id,
+                            "gfw_url": integration.base_url,
+                        },
+                    )
+                    return [msg]
+
+                response_per_type = []
+                if action_config.include_fire_alerts:
+                    fire_alerts_task = asyncio.create_task(
+                        client.get_fire_alerts(
+                            aoi_data=aoi_data,
+                            integration=integration,
+                            config=action_config
+                        )
+                    )
+                    fire_alerts = await fire_alerts_task
+                    if fire_alerts:
+                        logger.info(f"Fire alerts pulled with success.")
+                        transformed_data = [
+                            await transform_fire_alert(alert)
+                            for alert in fire_alerts
+                        ]
+                        response = await handle_transformed_data(
+                            transformed_data,
+                            str(integration.id),
+                            "pull_events"
+                        )
+                        response_per_type.append({"type": "fire_alerts", "response": response})
+
+                if action_config.include_integrated_alerts:
+                    integrated_alerts_task = asyncio.create_task(
+                        client.get_integrated_alerts(
+                            aoi_data=aoi_data,
+                            integration=integration,
+                            config=action_config
+                        )
+                    )
+                    integrated_alerts = await integrated_alerts_task
+                    if integrated_alerts:
+                        transformed_data = []
+                        logger.info(f"Integrated alerts pulled with success.")
+                        n = 0
+                        while n < len(integrated_alerts):
+                            transformed_integrated_alert = await transform_integrated_alert(integrated_alerts[n:n+4])
```

Comment:
@vgarcia13 this is not quite what I had in mind and I think we need to remove this. Apologies if my earlier comments were misleading.

I would like to simply create these Events individually as point events, just as we were doing before.

Later, in a separate story, we will come back and add a feature to create Geometry Events. I'll explain how to do that later.